src/bramble/client/main.js

```diff
@@ -920,6 +925,10 @@ define([
         this._executeRemoteCommand({commandCategory: "bramble", command: "BRAMBLE_DISABLE_WORD_WRAP"}, callback);
     };
 
+    BrambleProxy.prototype.configureAutoCloseTags = function(options, callback) {
+        this._executeRemoteCommand({commandCategory: "bramble", command: "BRAMBLE_CONFIGURE_AUTO_CLOSE_TAGS", args: [ options ]}, callback);
```

Comment:
this `args: [ options ]` feels a little out of place, and different from how all the other command bindings work. Is there a way for us to pass specifically the options needed and relevant to the auto-close-tags functionality?